src/Commands/BuildToolsCommand.php

```diff
@@ -658,9 +659,13 @@ protected function expandSourceAliases($source)
      */
     protected function autodetectUpstream($siteDir)
     {
-        return 'Empty Upstream';
-        // or 'Drupal 7' or 'WordPress'
-        // return 'Drupal 8';
+        if (file_exists($siteDir . '/web/wp-config.php')) {
```

Comment:
@stevector and @greg-1-anderson WordPress will first look for `wp-config.php` wherever core is. If WordPress core is in a subdirectory, say `wp`, then `web/wp/wp-config.php` will take precedence over `web/wp-config.php` if it exists.

That tripped me up when first implementing WordPress is a subdirectory via Composer as the Pantheon WordPress upstream includes `wp-config.php`. When installing core from our upstream repo into a subdirectory `web/wp/wp-config.php` must be deleted.

All that being said someone _could_ have `web/wp/wp-config.php`, or wherever they have installed WordPress core, without `web/wp-config.php` and have a properly working site. I think it's okay for us to be opinionated about using `wp` for the directory name as well as having `wp-config.php` live in the we docroot.